### DISCUSSION TITLE
Compatibility checks..

### DISCUSSION THREAD

**Discussion by joncage** [raomin/ESPAltherma#430] [Q&A] [answered]
I'm interested in integrating with my Daikin Atherma heat pump. We moved into our place (heat pump installed with the house) and have since added solar panels and a battery so quite interested in fine grained control possibilities in due course (one thing at a time though!).

Ours is a low-temperature-split unit with the indoor unit an EHVH11SU26CB6W - https://www.daikin.co.uk/content/dam/document-library/Installer-reference-guide/heat/air-to-water-heat-pump-low-temperature/EHVH11-16SU28CB6W,%20ERHQ-BV3,%20ERLQ-CV3_4PEN449979-1_Installer%20reference%20guide_English.pdf

When I first started looking at this project I hadn't used an ESP32 and hadn't realised there were different sticks out there but bought a JZK ESP-32S Dev board ( https://www.amazon.co.uk/ESP-32S-Development-2-4GHz-Bluetooth-Antenna/dp/B071JR9WS9 ) with the hopes of getting something up and running.

So, two questions:

1. Is my heatpump supported and if so, am I corerctly reading this that I should be using the `"def/Altherma(LT_CA_CB_04-08kW).h"` or possibly `//#include "def/Altherma(LT_CB_04-08kW Bizone).h"` ?
2. Is a JZK ESP-32S Dev board ( https://www.amazon.co.uk/ESP-32S-Development-2-4GHz-Bluetooth-Antenna/dp/B071JR9WS9 ) workable? ..and if so, is that just a standard ESP32 board to select?

**Comment by joncage** [comment]
Think I part-answered my second question as with a bit of fiddling and installing some silicon labs drivers for the USB <-> UART bridge I uploaded something...

```
Writing at 0x000cb50c... (96 %)
Writing at 0x000d0841... (100 %)
Wrote 809952 bytes (522585 compressed) at 0x00010000 in 46.4 seconds (effective 139.6 kbit/s)...
Hash of data verified.

Leaving...
Hard resetting via RTS pin...
=========================================================== [SUCCESS] Took 58.43 seconds ===========================================================

Environment    Status    Duration
-------------  --------  ------------
esp32          SUCCESS   00:00:58.431
=========================================================== 1 succeeded in 00:00:58.431 =========================================================== 
```

**Comment by raomin** [comment, marked as answer]
> Is my heatpump supported

Yes, your manual mentions X10A port.

> Is a JZK ESP-32S Dev board supported

Yes, it's an esp32. It's supported.
And yes, you properly uploaded the program, time for connection! :)
